app/assets/javascripts/beak/widgets/ractives/code-editor.coffee

```diff
@@ -74,6 +114,12 @@ window.RactiveModelCodeComponent = Ractive.extend({
                 on-click="recompile" {{# !isStale }}disabled{{/}} >Recompile Code</button>
           {{/}}
         </li>
+        <li class="netlogo-codetab-widget-listitem">
+          <input type='checkbox' class="netlogo-autocomplete-checkbox" checked='{{autoCompleteStatus}}'>
+          <label class="netlogo-autocomplete-label">
+            Auto Complete {{#if autoCompleteStatus}}Enabled{{else}}Disabled{{/if}}
```

Comment:
The standard way this is done throughout the rest of Galapagos is `Auto Complete {{# autoCompleteStatus}}Enabled{{else}}Disabled{{/}}` (that is, without the `if`s).  It's a small thing, but I'd prefer to see the `if`s removed.